.circleci/setupEnvironmentVars.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+INFILE=$1
+
+# get lines with a $, split by : and get the second field (i.e. where yaml value, the env variable name, is)
+# and clean it up by stripping whitespace, {, }, and "
+VARS=$(grep '\$' $INFILE | cut -d: -f2 | sed -e 's|[\$\{\}\"\s]||g')
+
+# uppercase the environment name and use it as suffix
+SUFFIX=${ENVIRONMENT^^}
```

Comment:
Just out of curiosity, what does the ^^ do? :)

Reply:
It uppercases the variable